ui/webpack.common.js

```diff
@@ -0,0 +1,105 @@
+const path = require('path');
+const HtmlWebPackPlugin = require('html-webpack-plugin');
+const CompressionPlugin = require('compression-webpack-plugin');
+const ESLintPlugin = require('eslint-webpack-plugin');
+const { DefinePlugin } = require('webpack');
+
+module.exports = (env) => ({
+  entry: {
+    metalk8s_ui: './src/index.js',
+  },
+  output: {
+    filename: 'static/js/[name].js',
+    path: path.resolve(__dirname, 'build'),
+    publicPath: '/',
+  },
+  resolve: {
+    modules: ['node_modules'],
+    extensions: ['.js', '.jsx', '.css', '.json'],
+    alias: {
+      _: [path.resolve(__dirname, 'public')],
+      'vega-lite': 'vega-lite/build/vega-lite.min.js'
+    },
+  },
+  /*
+   * https://webpack.js.org/configuration/performance/#performancehints
+   */
+  performance: {
+    hints: 'error',
+    // ~732 KiB for production
+    // ~4.06 MiB for development because flow increase the size of assets.
+    maxAssetSize: process.env.NODE_ENV === 'production' ? 750000 : 5000000,
+    assetFilter: (assetFilename) => {
+      return (
+        !assetFilename.endsWith('.map.gz') && assetFilename.endsWith('.gz')
+      );
+    },
+  },
+  module: {
+    rules: 
+    [
+      {
+        test: /\.(js|jsx)$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+        },
+      },
+      {
+        test: /\.html$/,
+        use: [
+          {
+            loader: 'html-loader',
+          },
+        ],
+      },
+      {
+        test: /\.jpe?g$|\.gif$|\.png$|\.ttf$|\.eot$|\.svg$/,
```

Comment:
Slightly more compact :wink:

```suggestion
        test: /\.(jpe?g|gif|png|ttf|eot|svg)$/,
```

Reply:
Done in 7279d56